pybandits/simulation_cmab.py

```diff
@@ -261,7 +262,7 @@ def get_proportion_positive_reward(self):
             + str(i): (
                 self.results["action"].loc[(self.results["group"] == i) & (self.results["reward"] == 1)].value_counts()
                 / self.results["action"].loc[(self.results["group"] == i)].value_counts()
-            ).to_dict()
+            ).to_serializable_dict()
```

Comment:
What was the issue here?

Reply:
No bug. My bad as I refactored (to_serializable_dict was to_dict before I pushed the code).